src/VscPreviewExtension/Microsoft.DocAsCode.Dfm.VscPreview/DfmProcess.cs

```diff
@@ -14,21 +14,27 @@ internal class DfmProcess
         static void Main(string[] args)
         {
             DfmJsonTokenTreeServiceProvider dfmJsonTokenTreeServiceProvider = new DfmJsonTokenTreeServiceProvider();
-            IMarkdownService dfmMarkdownService = dfmJsonTokenTreeServiceProvider.CreateMarkdownService(new MarkdownServiceParameters());
+            IMarkdownService dfmMarkdownService =
+                dfmJsonTokenTreeServiceProvider.CreateMarkdownService(new MarkdownServiceParameters());
             while (true)
             {
                 try
                 {
                     string command = Console.ReadLine();
+                    string result;
                     switch (command.ToLower().Trim())
                     {
                         case "exit":
                             return;
-                        case "dfmmarkup":
-                            DfmMarkupReceiveContent();
+                        case "docfxpreview":
+                            string baseDir = Console.ReadLine();
```

Comment:
Console.Write some message before ReadLine so that the user knows what to input?